pkg/refactor/database/db_schema.sql

```diff
@@ -90,6 +90,33 @@ CREATE TABLE IF NOT EXISTS "secrets" (
     FOREIGN KEY ("namespace_id") REFERENCES "namespaces"("oid") ON DELETE CASCADE ON UPDATE CASCADE
 );
 
+DROP TABLE IF EXISTS "runtime_variables";
+CREATE TABLE IF NOT EXISTS "runtime_variables" (
+    "id" uuid,
```

Comment:
I don't think this needs to exist. "name" + ("namespace_id"|"workflow_id"|"instance_id") should be a unique compound key. I almost want to merge all three columns into just one... But then we couldn't do delete cascades?

Reply:
We discussed this on chat.